_sass/blocks/_revenues_subpage_steps.scss

```diff
@@ -0,0 +1,119 @@
+// Revenues Subpage Steps
+//
+// Diving Graphics and associated steps
+//
+// Styleguide blocks.revenues_subpage_steps
+
+$onshore-oil-primary-color: #534136;
+$onshore-oil-secondary-color: #c5403c;
+$offshore-oil-primary-color: #b3e4f9;
+$offshore-oil-secondary-color: $blue;
+$coal-primary-color: #534136;
+$coal-secondary-color: #c5403c;
+$minerals-primary-color: #534136;
+$minerals-secondary-color: #c5403c;
+$offshore-renewables-primary-color: #534136;
+$offshore-renewables-secondary-color: #0b92ce;
+$onshore-renewables-primary-color: #579469;
+$onshore-renewables-secondary-color: $blue;
+
+@import 'revenues_subpage_diving_graphics';
+
+.revenues_subpage-dyk {
+  position: relative;
+  border-radius: $base-border-radius;
+  margin-bottom: $base-padding;
+  overflow: hidden;
+
+  @include respond-to(medium-up) {
+    display: block;
+    margin-top: 40%;
+    @include span-columns(3);
+  }
+}
+
+.revenues_subpage-dyk_heading {
+  color: $white;
+  background-color: $offshore-oil-secondary-color;
+  padding: $base-padding;
+  margin: 0;
+  text-align: center;
+}
+
+.revenues_subpage-dyk_text {
+  background-color: $white;
+  margin: 0;
+  padding: $base-padding;
+  padding-bottom: $base-padding-extra;
+}
+
+.revenues_subpage_dyk-link {
+  position: absolute;
+  right: $base-padding;
+  bottom: $base-padding;
+  span {
+    @include heading(h4);
+    line-height: $base-line-height;
+    vertical-align: bottom;
+  }
+}
+
+
+.revenues_subpage-steps_group {
```

Comment:
Selector `revenues_subpage-steps_group` should be written in lowercase with hyphens